inspirehep/modules/workflows/mappings/holdingpen/hep.json

```diff
@@ -23,6 +23,11 @@
                                 }
                             },
                             "type": "object"
+                        },
+                        "record_matches": {
+                            "include_in_all": false,
+                            "enabled": false,
+                            "type": "object"
```

Comment:
I had to specify `object` even if the new type is `integer` otherwise I wasn't able to use `enabled` property. Note that if I was specifying the `type` `integer` then I would have had to use `"index": "no"`... but... it would have still refused to index `objects`. Instead right now seems that ES is truly ignoring the type I am passing in.